src/docs/product/sentry-basics/guides/discover/index.mdx

```diff
@@ -103,6 +103,12 @@ Each table cell offers a dynamic context menu that allows you to easily continue
 
 ![Context menu](013.png)
 
+### Errors by Hour or Day
+
+The `timestamp.to_hour` and `timestamp.to_day` fields round down the timestamp to the last UTC hour or day. For example, an event with a `timestamp` of Mar 12, 2021 2:26:50 PM UTC will have a `timestamp.to_hour` of Mar 12, 2021 2:00:00 PM UTC and `timestamp.to_day` of Mar 12, 2021 12:00:00 AM UTC. This allows you to roll up events: for example, you can show hourly hit counts for errors hit at least twice per hour:
```

Comment:
```suggestion
The `timestamp.to_hour` and `timestamp.to_day` fields round down the timestamp to the last UTC hour or day respectively. For example, an event with a `timestamp` value of Mar 12, 2021 2:26:50 PM UTC, will have a `timestamp.to_hour` value of Mar 12, 2021 2:00:00 PM UTC and `timestamp.to_day` value of Mar 12, 2021 12:00:00 AM UTC. This allows you to roll up events. For example, you can show hourly hit counts for errors hit at least twice per hour:
```

Reply:
applied change sans comma. thank you for the suggestion.